planner/core/testdata/plan_suite_out.json

```diff
@@ -1195,17 +1195,17 @@
       },
       {
         "SQL": "select /*+ HASH_AGG() */ t1.a from t t1 where t1.a < any(select t2.b from t t2)",
-        "Best": "LeftHashJoin{IndexReader(Index(t.f)[[NULL,+inf]]->Sel([if(isnull(test.t.a), <nil>, 1)]))->TableReader(Table(t)->HashAgg)->HashAgg->Sel([ne(Column#27, 0)])}->Projection->Projection",
```

Comment:
After the second column pruning, the extra physical project can be eliminate in `postOptimize`.